doc/build-rpi.txt

```diff
@@ -0,0 +1,29 @@
+The following dependencies and steps are needed for Raspbian Stretch released in November 2017.
+
+Prequisites may change with newer versions.
+
+This has been tested with Raspberry Pi Zero W.
+
+Note that it may be tough to get everything + the blockchain installed on a 4GB flash card.  It's recommended you use an 8GB or 16GB flash card.
+
+
+#expand the default swapfile size to 1GB since 100MB is not enough
+$ sudo nano /etc/dphys-swapfile
+#change CONF_SWAPSIZE=100 to CONF_SWAPSIZE=1024
+$ sudo /etc/init.d/dphys-swapfile stop
+$ sudo /etc/init.d/dphys-swapfile start
```

Comment:
This probably could be replaced with `$ sudo service dphys-swapfile restart`, or `$ sudo systemctl restart dphys-swapfile`, don't know what init system Raspbian Stretch uses.

Reply:
Yes, likely so.  I didn't have time to optimize any of this at all.  I'm simply reporting what worked for me.  I wouldn't want to report steps I hadn't verified.  And I don't want to invest much more time into this right now.  But I agree with your sentiment here.